packages/toolkit/src/components/top-bar/NamespaceSwitch.tsx

```diff
@@ -372,13 +343,7 @@ export const NamespaceSwitch = () => {
       onOpenChange={(value) => {
         setSwitchIsOpen(value);
       }}
-      disabled={
-        env("NEXT_PUBLIC_APP_ENV") === "CLOUD"
-          ? namespacesRemainingCredit.isSuccess
-            ? false
-            : true
-          : false
-      }
+      disabled={false}
```

Comment:
You can just omit passing the disabled, so just remove it.

Reply:
addressed.